app/models.py

```diff
@@ -109,6 +109,35 @@ class DocumentRecommendation(db.Model):
     )
 
 
+class ModelSteps(db.Model):
+    __tablename__ = "ModelSteps"
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    type = db.Column(db.String(), nullable=False)
+
+
+class RecommendationModels(db.Model):
```

Comment:
```suggestion
class RecommendationModel(db.Model):
```
Alle Tabellen sind im singular